libcontainer/configs/intelrdt.go

```diff
@@ -10,4 +10,7 @@ type IntelRdt struct {
 	// The unit of memory bandwidth is specified in "percentages" by
 	// default, and in "MBps" if MBA Software Controller is enabled.
 	MemBwSchema string `json:"memBwSchema,omitempty"`
+
+	// Monitoring flag that's indicated to monitor llc-occupancy or total and local memory bandwidth.
```

Comment:
How about:
"// The flag to indicate if Intel RDT monitoring features (CMT and MBM) are enabled."?